history-of-quakers/modernized/28-appendix.adoc

```diff
@@ -526,10 +525,11 @@ at length became his friends and favorers.
 [.asterism]
 '''
```

Comment:
Not sure if this is redundant with how the spacing looks, but it was in the original so I left it for you to decide.

Reply:
I think it's ok how it is. 👍